src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -1366,6 +1366,31 @@ public static EdmTerm AddTerm(this EdmModel model, string namespaceName, string
             return term;
         }
 
+        /// <summary>
+        /// Resolves the annotations target and returns the target segments as Edm elements.
+        /// </summary>
+        /// <param name="model">The EdmModel.</param>
+        /// <param name="target">Target string containing segments separated by '/'. For example: "A.B/C/D.E/Func1(NS.T,NS.T2)/P1".</param>
+        /// <returns>The created enumeration of <see cref="IEdmElement"/>.</returns>
+        public static IEnumerable<IEdmElement> ResolveAnnotationsTarget(this IEdmModel model, string target)

```

Comment:
>(this IEdmModel model, string target) [](http://example.com/codeflow?start=71&length=37)

consider whether to add a third parameter about case-sensitive or not for the target path string?